src/page/Login/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  Flex,
+  Box,
+  Heading,
+  FormControl,
+  FormLabel,
+  Input,
+  Button,
+  Center,
+  Link
+} from '@chakra-ui/react';
+import Footer from '../../component/footer'
+
+export default function Login() {
+    return (
+    <div>
+    <Center h="100vh">
+      <Flex width="full" align="center" justifyContent="center">
+        <Box p={2}>
+            <Box textAlign="center">
+            <Heading as="h1" size="lg" letterSpacing={"10px"}>
+                AS<span style={{color: "#2b51cb"}}>TI</span><span style={{letterSpacing: "2px"}}>(Atasi <span style={{color: "#2b51cb"}}>Stunting</span>)</span>

```

Comment:
This is hard to read, let's increase the readability by changing it into a multi-line
```suggestion
                AS
                <span style={{color: "#2b51cb"}}>
                  TI
                </span>
                <span style={{letterSpacing: "2px"}}>
                  (Atasi 
                    <span style={{color: "#2b51cb"}}>
                       Stunting
                     </span>
                    )
                   </span>

```